themes/userstyles/layouts/styles/single.html

```diff
@@ -9,8 +9,13 @@
         <h1 class="tac">{{ .Params.name }}
             <span class="fg2"> by {{ .Params.author }}</span>
         </h1>
-        <img src="{{ .Params.preview }}"
-             alt="{{ .Params.name }} screenshot"/>
+        <div class="card col gap animate ">
```

Comment:
`col gap` classes aren't needed, but you'll need to reset padding. Try adding a `p0` class in `main.scss` with `padding: 0`.
```suggestion
        <div class="card animate p0">
```

Reply:
Haha I just copied over the classes from the cards in the hompage and I forgot to remove the extra ones :)